cms/templates/product_page.html

```diff
@@ -10,4 +10,5 @@
 {% endblock %}
 {% block content %}
   <div>{{ page.description|safe}} </div>
+  {% if faculty %} {% include "partials/faculty-carousel.html" with page=faculty_data %} {% endif %}
```

Comment:
What's `faculty_data` here? I don't see it defined anywhere else.

Reply:
It coming from `get_context` method.

Reply:
There's no need to duplicate that when we already have the page available in `page.faculty`

Reply:
The page context is not accessible on the template so that's why used that faculty_data for testing it.

Need to debug it that why the page data is not accessible in the templates.